hi-ml-histopathology/testhisto/testhisto/models/test_deepmil.py

```diff
@@ -159,7 +169,8 @@ def mock_panda_slides_root_dir(tmp_path_factory: pytest.TempPathFactory) -> Path
         tiles_pos_type=TilesPositioningType.RANDOM
     )
```

Comment:
To align with `mock_panda_tiles_root_dir` should we add a similar log here that the slides are being generated (and will be deleted after the test)?

Reply:
yes, will do!